schemas/threatest.schema.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/datadog/threatest/threatest.schema.json",
+  "title": "Threatest Test Suite",
+  "description": "Schema for a Threatest test suite",
+  "type": "object",
+  "required": [
+    "scenarios"
+  ],
+  "properties": {
+    "scenarios": {
+      "description": "The display name of the vulnerability",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "The list of scenarios",
+        "required": [
+          "description",
```

Comment:
Description sounds to me a bit like a documentation field which should eventually be set. What do you think of using `name` instead?

Reply:
done in aaf8c50798dedec8651fb885dae957bad21e1508 and added a unit test for the parser in e5a96f488c4b22665cf2bcd39489beb1bf69403a